grafana-plugin/src/styles/utils.styles.ts

```diff
@@ -24,6 +29,34 @@ export const getUtilStyles = (_theme: GrafanaTheme2) => {
   };
 };
 
+export function getLabelBackgroundTextColorObject(
```

Comment:
This part is actually the core of `Badge` component in grafana but changing `Tag` to be replaced to `Badge` in all our usecases would have taken a lot more time, and instead I preferred this approach where we rely on tinycolor to generate these (same as grafana)

Reply:
is there any helper in `@grafana/ui` package that returns such object? I think it could be there...

Reply:
I found several helpers: `getTagColorsFromName, getTextColorForBackground, getTextColorForAlphaBackground` etc, maybe the can be used here

Reply:
Neither of them is closer to what this does :(